src/events/NewYear.ts

```diff
@@ -21,6 +22,9 @@ export default defineEvent({
     if (numOfLaws < 3) {
       return 0
     }
+    if (numOfLaws > 4) {
```

Comment:
This is not the inverse condition and contradicts the probability calculation below which counts 3 to 5 events to make this event possible.

Reply:
The idea is that after 5 laws the event shall happen for certain. Maybe `numOfLaws >= 5` would have made that more apparent?

Reply:
Ah, ok, I didn't get it right.